google/cloud/bigquery/schema.py

```diff
@@ -291,3 +329,72 @@ def _to_schema_fields(schema):
         field if isinstance(field, SchemaField) else SchemaField.from_api_repr(field)
         for field in schema
     ]
+
+
+class PolicyTagList(object):
+    """Define Policy Tags for a column.
+
+    Args:
+        names (Optional[Tuple[str]]): list of policy tags to associate with
+            the column.
+    """
+
+    def __init__(self, names=None):
+        self._properties = {}
+        self.names = names
+
+    @property
+    def names(self):
+        """Tuple[str]: Policy tags associated with this definition.
+        """
+        return self._properties.get("names", ())
+
+    @names.setter
+    def names(self, value):
+        """Optional[Tuple[str]]: Policy tags associated with this definition.
+
+        (Defaults to :data:`None`).
+        """
+        if value is not None:
+            self._properties["names"] = value
+        else:
+            if "names" in self._properties:
+                del self._properties["names"]
+
+    @classmethod
+    def from_api_repr(cls, api_repr):
+        """Return a :class:`PolicyTagList` object deserialized from a dict.
+
+        This method creates a new ``PolicyTagList`` instance that points to
+        the ``api_repr`` parameter as its internal properties dict. This means
+        that when a ``PolicyTagList`` instance is stored as a property of
+        another object, any changes made at the higher level will also appear
+        here.
+
+        Args:
+            api_repr (Mapping[str, str]):
+                The serialized representation of the PolicyTagList, such as
+                what is output by :meth:`to_api_repr`.
+
+        Returns:
+            google.cloud.bigquery.schema.PolicyTagList:
+                The ``PolicyTagList`` object.
+        """
+        instance = cls()
+        instance._properties = copy.deepcopy(api_repr)
+        return instance
+
+    def to_api_repr(self):
+        """Return a dictionary representing this object.
+
+        This method returns the properties dict of the ``PolicyTagList``
+        instance rather than making a copy. This means that when a
+        ``PolicyTagList`` instance is stored as a property of another
+        object, any changes made at the higher level will also appear here.
+
+        Returns:
+            dict:
+                A dictionary representing the PolicyTagList object in
+                serialized form.
+        """
+        return self._properties
```

Comment:
```suggestion
        return copy.deepcopy(self._properties)
```
We should return a copy here as well.